docker-compose.yml

```diff
@@ -1,5 +1,40 @@
 version: '2'
 services:
+  antispam:
+    image: mailu/rspamd:stable
+    restart: always
+    volumes:
+      - /var/lib/rspamd
+
+  antivirus:
+    image: mailu/clamav:stable
+    restart: always
+    volumes:
+      - /data
+
+  certbot:
+    build: certbot
+    restart: always
+    env_file: env
+    environment:
+      CERTBOT_DOMAINS: |-
+        app.foodcoops.net
+        foodcoops.net,www..foodcoops.net
+      CERTBOT_EMAIL: admin@foodcoops.net
+      CERTBOT_TOUCH_FILE: /haproxy_triggers/reload
```

Comment:
At the other places we use the list style, let's do that here as well.

Reply:
yes, but the other style does not support multilines, which makes the list of certificate domains MUCH MORE readable. especially, if you have a longer list (e.g. custom domains of foodcoops)

Reply:
Ah, that makes sense. :+1: